explanation/epa-orchestrator.rst

```diff
@@ -0,0 +1,46 @@
+CPU Pinning
+================
+
+Introduction
+------------
+
+CPU pinning is the binding of specific processes to dedicated CPU cores. The `isolcpus` kernel command line parameter is one way of configured isolated CPUs for the node. Once set, the system lists the isolated cores at `/sys/devices/system/cpu/isolated` on a given node.
```

Comment:
Use double backquotes for inline formating: https://canonical-starter-pack.readthedocs-hosted.com/latest/reference/rst-syntax-reference/
`isolcpus` => ``isolcpus``
`/sys/devices/system/cpu/isolated` => ``/sys/devices/system/cpu/isolated``

(and all other cases below)